web_src/js/components/RepoActionView.vue

```diff
@@ -884,15 +892,18 @@ export function initRepositoryActionView() {
   border-radius: 0;
 }
 
-/* TODO: group support
-
-.job-log-group {
-
+.job-log-group .job-log-list .job-log-line .log-msg {
+  margin-left: 2em;
 }
-.job-log-group-summary {
 
+.job-log-group-summary {
+  position: relative;
 }
-.job-log-list {
 
-} */
+.job-log-group-summary > .job-log-line {
+  position: absolute;
+  inset: 0;
+  z-index: -1;
```

Comment:
What problem is this negative z-index solving?

Reply:
To avoid hiding this:

![image](https://github.com/user-attachments/assets/bb88b43d-05f8-47ff-991c-587db4f00f5f)

Reply:
```suggestion
  z-index: -1; /* to avoid hiding the triangle of the "details" element */
```